### PR TITLE
[ricos-schema] gif schema

### PR DESCRIPTION
**Pull request by maimonav**
Giphy ==> Gif (only in schema).

### CODE REVIEW

File: packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/plugin_gif.proto [hunk 1]
```diff
@@ -5,7 +5,7 @@ package wix.rich_content.v1;
 import "wix/rich_content/v1/common.proto";
 import "google/protobuf/wrappers.proto";
 
-message GiphyData {
+message GifData {
     PluginContainerData container_data = 1;
     GIFSource gif = 2;
 }
```
Comment (mharsat) [inline packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/plugin_gif.proto:11]: Let's think if we want to make changes to `GIFSource` too before finalizing

**Review by mharsat — CHANGES_REQUESTED** [review]

File: packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/plugin_gif.proto (hunk 1 above)
Comment (maimonav) [reply packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/plugin_gif.proto:11]: What changes do you have in mind?

**Review by shaulgo — CHANGES_REQUESTED** [review]

File: packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/plugin_gif.proto (hunk 1 above)
Comment (mharsat) [reply packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/plugin_gif.proto]: @shaulgo and I were thinking of reorganizing the different GIF URLs. Maybe using the common `Media` message

[the remark this answers is not in the document]
Comment (mharsat) [reply packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/plugin_gif.proto:11]: And then have a `source` and `downsizedSource`?

[the remark this answers is not in the document]
Comment (maimonav) [reply packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/plugin_gif.proto:14]: Please take a look @mharsat @shaulgo

File: packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/plugin_gif.proto [hunk 2]
```diff
@@ -7,16 +7,14 @@ import "google/protobuf/wrappers.proto";
...
-    string still_url = 4;
-    google.protobuf.StringValue original_mp4 = 5;
-    google.protobuf.StringValue downsized_url = 6;
-    google.protobuf.StringValue downsized_still_url = 7;
-    google.protobuf.StringValue downsized_small_mp4 = 8;
-}
+message GIF {
+    google.protobuf.StringValue gif = 1;
+    google.protobuf.StringValue mp4 = 2;
+    google.protobuf.StringValue still = 3;
```
Comment (mharsat) [inline packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/plugin_gif.proto:19]: Lat's have some fun with validations 🚀

```suggestion
    google.protobuf.StringValue gif = 1 [(wix.api.format) = WEB_URL];
    google.protobuf.StringValue mp4 = 2 [(wix.api.format) = WEB_URL];
    google.protobuf.StringValue still = 3 [(wix.api.format) = WEB_URL];
```

**Review by mharsat — APPROVED** [review]
Looks great!

File: packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/plugin_gif.proto (hunk 2 above)
Comment (mharsat) [reply packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/plugin_gif.proto:19]: It's not trivial that gif/mp4 are URLs here, so the validation can help.
Also documenting `GIFData` would be divine

Comment (maimonav) [reply packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/plugin_gif.proto:19]: Is this Wix public api? (open source concern)

Comment (mharsat) [reply packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/plugin_gif.proto:19]: Nope, it's removed in the generated open source proto

Comment (mharsat) [reply packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/plugin_gif.proto:19]: https://github.com/wix/ricos/blob/4986671018e89b544344771815287bd1367cdfd7/packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/rich_content.proto#L128-L129

**Pull request merged**